packages/google_maps_flutter/google_maps_flutter_web/example/integration_test/projection_test.dart

```diff
@@ -75,7 +74,6 @@ void main() {
         await pumpCenteredMap(
           tester,
           initialCamera: initialCamera,
-          size: size,
```

Comment:
This is another situation where we are passing a variable we set as the default value. I think we would be better off removing the default size from `pumpCenteredMap`.

Reply:
fixed in [38eaffa](https://github.com/flutter/plugins/pull/6163/commits/38eaffa10e34d442ad98669b697221ea513615a4)